recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -109,13 +109,13 @@ do_install_append () {
         printf "[provision]\nprimary_ecu_hardware_id = ${SOTA_HARDWARE_ID}\n" > ${D}${libdir}/sota/conf.d/40-hardware-id.toml
     fi
 
-    if [ -n "${SOTA_SECONDARY_CONFIG_DIR}" ]; then
-        if [ -d "${SOTA_SECONDARY_CONFIG_DIR}" ]; then
+    if [ -n "${SOTA_SECONDARY_CONFIG}" ]; then
```

Comment:
Looks like it would break virtual secondaries on yocto images. As far as I know we still use it, at least for QA.

Reply:
Yes, you're right. I'll roll back usage of SOTA_SECONDARY_CONFIG_DIR and just introduce a new configuration parameter SOTA_SECONDARY_CONFIG. As result, both of them can be used in parallel until the virtual secondary configuration is not moved to the new approach, so SOTA_SECONDARY_CONFIG_DIR can be removed once it happens.

Reply:
Updated